lib/matplotlib/pyplot.py

```diff
@@ -706,11 +706,11 @@ def xkcd(
     scale: float = 1, length: float = 100, randomness: float = 2
 ) -> ExitStack:
     """
-    Turn on `xkcd <https://xkcd.com/>`_ sketch-style drawing mode.  This will
-    only have effect on things drawn after this function is called.
+    Turn on `xkcd <https://xkcd.com/>`_ sketch-style drawing mode. This will
+    only have an effect on things drawn after this function is called.
```

Comment:
Since this line is changing, it really should be split into two paragraphs.
```suggestion
    Turn on `xkcd <https://xkcd.com/>`_ sketch-style drawing mode.
    
    This will only have an effect on things drawn after this function is called.
```